hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -50,6 +50,47 @@
       milestone: V2
       tests: ["pattgen_perf"]
     }   
+    {
+      name: cnt_rollover
+      desc: '''
+            Checking ip operation with random counter values
+
+            Stimulus:
+              - Program the pre-divider and size registers to unconstraint random values
+              - Program the clk_cnt, bit_cnt and rep_cnt to values less than but close to
+		predivider and size registers, so that counting would take a reasonable
+		number of clock cycles
+	      - include programming for corner cases
```

Comment:
Hi Alex, I can confirm that you have tabs in your submission.  The picture below is from emacs' whitespace-mode. 
![image](https://user-images.githubusercontent.com/47870387/132744398-52a81da8-a177-4b21-9f5e-324f78334d17.png)

You can automatically eliminate these with the `expand` command.  e.g.:
```
expand pattgen_testplan.hjson > tmp
mv tmp pattgen_testplan.hjson
```
You also have some trailing spaces which are not allowed.

Reply:
I eliminated tabs and trailing spaces.

Reply:
Thanks,